cluster-agent/main.go

```diff
@@ -112,4 +119,9 @@ func main() {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
+
+	//Retrieving ArgoCD metrics: Application Controller, API Server, Repo Server
```

Comment:
These three lines can be removed for now (we'll add code that calls these functions to the appropriate spot with later stories).